login.css

```diff
@@ -0,0 +1,5 @@
+.disclaimer{
+    font-size: 1.5rem;
+    text-align: center;
```

Comment:
Make a class for stuff like this so you can re use it wherever you want.

Reply:
That's a personal preference so maybe I will do it later.